backend/tests/fixtures/llm_responses.py

```
@@ -139,3 +139,110 @@ def mock_chat_response(content: str, tool_calls=None) -> ChatResponse:
+    """モックAnthropicストリーミングレスポンスの行を生成します。
+
+    Anthropicは改行区切りのJSONを返します。
+
+    Args:
+        text_chunks: テキストチャンクのリスト
+
+    Returns:
+        Anthropic SSE形式の行リスト
+    """
+    lines = []
+    for i, chunk in enumerate(text_chunks):
```

Ruff (0.14.13)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)